tests/src/oracle_tests.rs

```diff
@@ -94,16 +99,16 @@ fn register_feeder() {
     let _user = Addr::unchecked(USER);
     let _admin = Addr::unchecked(ADMIN);
 
-    oracle_wrapper::add_feeder(&mut test_case, ADMIN);
+    add_feeder(&mut test_case, ADMIN);
 }
 
 #[test]
 fn internal_test_integration_setup_test() {
     let mut test_case = create_test_case();
 
-    oracle_wrapper::add_feeder(&mut test_case, ADMIN);
+    add_feeder(&mut test_case, ADMIN);
```

Comment:
[many occurences] What's the motivation for using the functions by name only? Rust idioms recommend other

Reply:
Interference between the crate and the module, because now that both are with the same name it causes problems when trying to distinguish it.